packages/botonic-plugin-flow-builder/src/content-fields/hubtype-fields/knowledge-base.ts

```diff
@@ -5,6 +5,9 @@ export interface HtKnowledgeBaseNode extends HtBaseNode {
   type: HtNodeWithContentType.KNOWLEDGE_BASE
   content: {
     feedback_enabled: boolean
-    sources: string[]
+    sources_data: {
```

Comment:
do you think it's worth to have an interface `SourceData`? is this type going to change a lot? if not it's ok like this, but maybe it's better to have it defined in an interface